src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,61 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import { Livechat } from '../../../api';
+import I18n from '../../../i18n';
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import styles from './styles.scss';
+
+
+const DisplayCallIframe = (rid) => (
+	<Screen.Content nopadding>
+		<div className={createClassName(styles, 'iframe')}>
+			<iframe style='height:100%' src={`${ Livechat.client.host }/meet/${ rid.rid }`} />
+		</div>
+	</Screen.Content>
+);
+
+
+export const CallNotification = (props) => {
+	const [isframe, setIframe] = useState(false);
+	const [show, setShow] = useState(true);
+
+	const acceptClick = async () => {
+		setShow(!{ show });
+		if (props.rid.t === 'jitsi_call_started') {
+			try {
+				const result = await Livechat.videoCall(props.rid);
+				window.open(`https://${ result.videoCall.domain }/${ result.videoCall.room }`);
+				return;
+			} catch (error) {
+				console.error(error);
```

Comment:
There's some room for improvement here. Currently we're only logging the error in console which is not very user friendly UX. A good UX would be to show the error message like [this](https://user-images.githubusercontent.com/34130764/125165846-587fcc80-e1b6-11eb-88a4-9736c154ba77.png)